jmclient/jmclient/schedule.py

```diff
@@ -65,17 +69,24 @@ def get_amount_fractions(power, count):
     setting, so we make sure it's appreciable to begin with.
     """
     while True:
-        amount_fractions = rand_pow_array(power, count)
-        amount_fractions = [1.0 - x for x in amount_fractions]
-        y = [x / sum(amount_fractions) for x in amount_fractions]
+        knives = [random.random() for i in range(count-1)]
+        knives = sorted(knives)[::-1]
+        y = []
+        l = 1
+        k = 1
```

Comment:
Why this? You're about to iterate over `knives` with `k` on the next line. Is there a use-case for `count=0`?

Reply:
There's a test somewhere which tests with count=0 I think, i added that to stop the test failing